From 13e3f51e0642284c248185a35936d53364f3c63a Mon Sep 17 00:00:00 2001
From: Jan Engelhardt <jengelh@inai.de>
Date: Thu, 2 Apr 2015 12:44:23 +0200
Subject: [PATCH 40/61] plugins: use cached copy of plugin description list at
 iridiumbrowsing.de

The list contains mimetype-to-plugin mappins, as well as blacklists
for security-impeded old versions of plugins.

To avoid contacing Google, let the UA use the plugin list hosted at
iridiumbrowser. The Iridium project re-gathers these lists on a
regular basis.
---
 chrome/browser/plugins/plugins_resource_service.cc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/chrome/browser/plugins/plugins_resource_service.cc b/chrome/browser/plugins/plugins_resource_service.cc
index 1e063a0c1aab3..2cea285d0e72f 100644
--- a/chrome/browser/plugins/plugins_resource_service.cc
+++ b/chrome/browser/plugins/plugins_resource_service.cc
@@ -54,7 +54,7 @@ const int kStartResourceFetchDelayMs = 60 * 1000;
 const int kCacheUpdateDelayMs = 24 * 60 * 60 * 1000;
 
 const char kPluginsServerUrl[] =
-    "https://www.gstatic.com/chrome/config/plugins_3/";
+    "https://cache.iridiumbrowser.de/plugins_3/";
 
 GURL GetPluginsServerURL() {
   std::string filename;
-- 
2.26.0

